code/controllers/subsystem/ticker.dm

```diff
@@ -30,8 +30,7 @@ SUBSYSTEM_DEF(ticker)
 	var/list/availablefactions = list()		//list of factions with openings
 	var/list/scripture_states = list(SCRIPTURE_DRIVER = TRUE, \
 	SCRIPTURE_SCRIPT = FALSE, \
-	SCRIPTURE_APPLICATION = FALSE, \
-	SCRIPTURE_JUDGEMENT = FALSE) //list of clockcult scripture states for announcements
```

Comment:
why isn't this throwing an error if it's not defined anywhere

Reply:
It's a "feature" of DM.
Remember "unquoted list associations"?
It's turning ```SCRIPTURE_JUDGEMENT``` into ```"SCRIPTURE_JUDGEMENT"``` and assigning ```FALSE``` to it.

if ```SCRIPTURE_JUDGEMENT``` did exist, it would make that the key, but it doesn't, so it falls back to this """"feature""""

Reply:
See: https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use